tendrl/ceph_integration/manager/__init__.py

```diff
@@ -54,8 +54,6 @@ def main():
             "please Import or Create sds cluster in Tendrl "
             "and include Node %s" % NS.node_context.node_id
         )
-    
-    NS.tendrl_context.save()
```

Comment:
You still need to save the tendrl context @shtripat , dont remove from any service